app/Models/User.php

```diff
@@ -228,4 +230,24 @@ public function toSitemapTag(): Url
         return Url::create($this->profile_url)
             ->setChangeFrequency(Url::CHANGE_FREQUENCY_DAILY);
     }
+
+    public function toSearchableArray(): array
+    {
+        return [
+            'name' => $this->name,
+            'display_name' => $this->display_name,
+            'avatar_url' => $this->avatar_url,
+            'profile_url' => $this->profile_url,
```

Comment:
most probably you don't want the URLs to be searchable, isn't it?

Reply:
No, but as display attribute. Do it's not configured as searchable attribute on the index.
I thought that's the way to do display only attributes. 🤔

Reply:
By default all attributes will be available for display, this is a way to configure which ones are searchable.